services/slam/builtin/builtin.go

```diff
@@ -57,7 +57,7 @@ const (
 	parsePortMaxTimeoutSec      = 60
 	// TODO change time format to .Format(time.RFC3339Nano) https://viam.atlassian.net/browse/DATA-277
 	// time format for the slam service.
```

Comment:
[nit] Remove TODO. There are also a few other TODOs for DATA-277, so please make sure to remove them all.